libs/map/src/lib/components/action/vl-map-click-action.ts

```diff
@@ -0,0 +1,78 @@
+import {webComponent} from '@domg-wc/common-utilities';
+import {VlMap} from '@domg-wc/map';
+import {LitElement} from "lit";
+import {MapBrowserEvent} from "ol";
+import Overlay from "ol/Overlay";
+import {property} from 'lit/decorators.js';
+import {Coordinate} from "ol/coordinate";
+import {Extent} from "ol/extent";
+import {VlMapClickActionPindrop} from "./vl-map-click-action-pindrop";
+import {Pixel} from "ol/pixel";
+
+/**
+ * VlMapClickAction
+ * @classdesc The map click action component. Adds a marker on the map.
+ * @property {function} onClick - callback die afgevuurd wordt na click op map
+ */
+
+// TODO: crs doorsturen? vb: crs=EPSG:31370 => komt uit map??
```

Comment:
Weet niet of dit nu effectief nodig is... projectie is map gerelateerd? @driesva

Reply:
ik denk dat het aan de map experten (team Decibel?) is om hier een beter antwoord op te geven. Voorlopig is vl-map altijd Lamber72 dacht ik, dus nog niet nodig.
Mijn comment was eerder toekomstgericht.